internal/connector/internal/api/admin/private/api/openapi.yaml

```diff
@@ -133,6 +133,48 @@ paths:
       summary: Returns a list of connector clusters
       tags:
       - Connector Clusters Admin
+  /api/connector_mgmt/v1/admin/kafka_connector_clusters/{connector_cluster_id}:
```

Comment:
not quite sure where this came from. Connectors related change that appeared after `make generate`.

Reply:
This is an uncommited change from someone else's PR. Please undo changes not related to this PR.
This is a recurring issue. I created a Jira issue to deal with this https://issues.redhat.com/browse/MGDSTRM-8508 to try to prevent it from happening.

Reply:
seems to be something missing from here https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1016/